README.md

```diff
@@ -1,70 +1,65 @@
-* Work with datasets that exceed GPU and CPU memory without having to worry about scale.
-* Focus on what to do with the data, and not how to do it, using our abstraction at the operation level.
+* prepare datasets quickly and easily for experimentation so that more models can be trained
+* work with datasets that exceed GPU and CPU memory without having to worry about scale.
+* focus on what to do with the data, and not how to do it, using our abstraction at the operation level.
 
-It is also meant to help ML/Ops Engineers deploying models into production by providing:
+It is also meant to help ML/Ops Engineers with deploying models into production by providing faster dataset transformation. This only makes it easy for production models to be trained more frequently and kept up to date, helping improve responsiveness and model performance.
 
-* Faster dataset transformation, allowing for production models to be trained more frequently and kept up to date, helping improve responsiveness and model performance.
-* Integration with model serving frameworks like [NVIDIA’s Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) and [Tensorflow Serving](https://github.com/tensorflow/serving) to make model deployment easy. (WIP)
-* Statistical monitoring of the dataset for distributional shift and outlier detection during production training or inference. (WIP)
+The library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset. Loading tabular data is almost always a training bottleneck. In our benchmarking, we've seen 10x improvements in training time on the GPU relative to the native dataloaders.
 
-The library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset.  Loading tabular data is almost always a training bottleneck and in our benchmarking we have seen 10x improvements in training time on GPU relative to the native dataloaders. 
+Extending beyond model training, we plan on integrating with model-serving frameworks like NVIDIA’s Triton Inference Server. This will only create a clear path to production inference for these models and allow the feature engineering and preprocessing steps performed on the data during training to be automatically applied to incoming data during inference.
```

Comment:
```suggestion
Extending beyond model training, we plan on integrating with model-serving frameworks like NVIDIA’s Triton Inference Server. This will create a clear path to production inference for these models and allow the feature engineering and preprocessing steps performed on the data during training to be automatically applied to incoming data during inference.
```